coding-concepts.js

```diff
@@ -67,7 +73,9 @@ class Learn {
   }
 }
 var learnStudent = new Learn("George")
-// console.log(learnStudent)
+console.log(learnStudent)
 
 // a) Your answer:
-// b) Verify and explain:
+// Learn { student: "George", cohort: "Bravo", year: 2022 }
+
+// b) Verify and explain: Learn { student: 'George', cohort: 'Bravo', year: 2022 } - Here we have our template or class called "Learn", so Line 75 is asking from our previously established template, if "George" is plugged in, what would be the output? - which in this case would be the answer/output previously stated or found above. 
```

Comment:
Awesome job, Steven!  You did great in this section!  Excellent work and explanations!